packages/api/src/controllers/stripe.ts

```diff
@@ -82,7 +82,8 @@ export const reportUsage = async (req: Request, adminToken: string) => {
 async function reportUsageForUser(
   req: Request,
   user: WithID<User>,
-  adminToken: string
+  adminToken: string,
+  actuallyReport: boolean = true
```

Comment:
ts tip: `: boolean` is auto inferred when you add the default value
```suggestion
  actuallyReport = true
```